content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -87,51 +141,54 @@ To retrieve data from Filecoin using Lassie, all you need is the CID of the cont
     ...
     ```
 
-1. This will create an `output.car` file within your current directory:
+* `-o` is an optional flag that tells Lassie where to write the output to. If you don't specify a file, it will append `.car` to your CID and use that as the output file name.
 
-    ```shell
-    ls -l
-    ```
+If you specify `-`, as in our above example, the output will be written to `stdout` so it can be piped to another command, such as `go-car`, or redirected to a file.
 
-    ```shell
-    total 143M
-    -rw-rw-r-- 1 user user 143M Feb 16 11:21 output.car
-    ```
+* `<CID>/path/to/content` is the CID of the content you want to retrieve, and an optional path to a specific file within that content.
 
-### Extract data
+A CID is always necessary and, if you don't specify a path, Lassie will attempt to download the entire content. If you specify a path, Lassie will only download that specific file or, if it is a directory, the entire directory and its contents.
 
-Now that we’ve downloaded a CAR file, we need to find out what’s inside it. 
+##### go-car CLI usage
 
-1. The format for extracting a `.car` file using Go-car is:
+The `car extract` command can be used to extract files and directories from a CAR:
 
-    ```shell
-    car extract --file <INPUT_FILE>
-    ```
+```shell
+car extract -f <INPUT_FILE>[/path/to/file/or/directory] [<OUTPUT_DIR>]
+```
 
-1. Extract the `output.car` file we just downloaded using Lassie:
+* `-f` is an optional flag that tells `go-car` where to read the input from. If omitted, it will read from `stdin`, as in our example above where we piped `lassie fetch -o -` output to `car extract`.
 
-    ```shell
-    car extract --file output.car
-    ```
+* `/path/to/file/or/directory` is an optional path to a specific file or directory within the CAR. If omitted, it will attempt to extract the entire CAR.
 
-    This command does not output anything on success.
+* `<OUTPUT_DIR>` is an optional argument that tells `go-car` where to write the output to. If omitted, it will write to the current directory.
 
-1. You can list the output of the `car` command with `ls`:
+If you supply `-`, as in the above example, it will attempt to extract the content directly to `stdout`. This will only work if we are extracting a single file.
 
-    ```shell
-    ls -lh
-    ```
+  In the example above, the `>` operator was used to redirect the output of `car extract` to a named file,. This is because the content we fetched was raw file data that did not have a name encoded. In this case, if we didn't use `-` and `> filename`, `go-car` would write to a file named `unknown`.
```

Comment:
```suggestion
  In the example above where we fetched a file named `lidar-data.tar`, the `>` operator was used to redirect the output of `car extract` to a named file,. This is because the content we fetched was raw file data that did not have a name encoded. In this case, if we didn't use `-` and `> filename`, `go-car` would write to a file named `unknown`. In this instance `go-car` was used to reconstitute the file from the raw blocks contained within Lassie's CAR output.
```

Take or leave that second part, my main concern was "example above" because we have >1 of them above. But my point with the second bit is to try and address the confusion that will inevitably arise from "why isn't Lassie just spitting out my file?", it's quite confusing for people when a single file isn't a single block, or when it is a single block, why that block isn't just the file itself. You have to "reconstitute" / "reify" the file back from raw IPLD form and in the case of a large .tar file, that's a _lot_ of blocks to piece back together.